www/js/controllers.js

```diff
@@ -5,8 +5,7 @@ angular.module('emission.controllers', ['emission.splash.updatecheck',
                                         'emission.splash.pushnotify',
                                         'emission.splash.localnotify',
                                         'emission.survey.launch',
-                                        'emission.stats.clientstats',
-                                        'emission.incident.posttrip.prompt'])
+                                        'emission.stats.clientstats',])
```

Comment:
does this work syntactically in spite of the trailing comma?

Reply:
Yes, trailing comma is syntactically corrected.

Reply:
I would normally do this since it helps avoid potential bugs.